pages/execdashboard.js

```diff
@@ -111,7 +138,16 @@ const ExecDashboard = () => {
                         <AddMeetingForm/>
                         <br></br>
                         <h2 className={styles.header}>View Meeting Attendance</h2>
-                        <SelectMeeting selectMeeting={getAttendees}/>
+                        <SubmitNotification showNotif={showNotif} setShowNotif={setShowNotif}/> 
+                        {refresh && <SelectMeeting selectMeeting={getAttendees}/>}
```

Comment:
is there a reason for refresh to be `null` and `1` instead of `false` and `true`